### PR TITLE
Add Comet v0.7 into PHP frameworks

### LINKED ISSUE #2942: [PHP] Add Comet
There is a relatively new PHP Comet framework, might be useful to add it to test set as well 

https://github.com/gotzmann/comet

### PR DESCRIPTION
**Pull request by gotzmann**
Would like to see Comet between other PHP frameworks :)

Original framework: https://github.com/gotzmann/comet

---------
Closes #2942

### CODE REVIEW

File: php/comet/composer.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "require": {
+    "gotzmann/comet": "^0.7.0"
```
Comment (waghanza) [inline php/comet/composer.json:3]: could you please use `~` notation

File: php/comet/start.php [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<?php
+require_once __DIR__ . '/vendor/autoload.php';
+
+use Comet\Comet;
+
+$app = new Comet([ 
+	'port' => 3000,
+	shell_exec('nproc') ?? 32,
```
Comment (waghanza) [inline php/comet/start.php:8]: rather use environment variable (like for `swoole` based **frameworks**, simps, hyperf ...)

**Comment by waghanza** [comment]
`CI` is failing but it's on my end. I'll merge and update the result for this saturday

**Comment by waghanza** [comment]
May I add you to `CODEOWNERS` @gotzmann ? so you 'll be noticed for each `comet` changes `here`

**Comment by gotzmann** [comment]
> May I add you to `CODEOWNERS` @gotzmann ? so you 'll be noticed for each `comet` changes `here`

Yes of course!

**Pull request merged**